.gitignore

```diff
@@ -165,6 +165,7 @@ cmake_install.cmake
 /lib/cpluff/loader/cpluff-loader
 /lib/cpluff/m4/
 /lib/cpluff/stamp-h1
+/lib/cpluff/tmp*
```

Comment:
unrelated to this PR, if this is needed move it at least into it's own commit (better would be a separate PR)